models/beer_info.py

```diff
@@ -0,0 +1,14 @@
+from .beer import Beer
+
+
+class BeerInfo(Beer):
+    def __init__(self, beer):
+        self.name = beer.name
```

Comment:
Jak dziedziczymy po Beer to można chyba tak: 
```python
def __init__(self, beer):
     super().__init__(beer.name, beer.link, beer.explanation)
     self.count = 0
```

Reply:
słuszna uwaga